File: Backend/src/ShoeStore.Application/DTOs/ProfileDTOs/ResponseProfileDto.cs
namespace ShoeStore.Application.DTOs.ProfileDTOs;

public class ResponseProfileDto
{
    public Guid UserGuid { get; set; }
    public string UserName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;

    public string AvatarUrl { get; set; } = string.Empty;
    public DateTime? DateOfBirth { get; set; }
}

File: Frontend/app/src/main/java/com/example/shoestoreapp/features/user/profile/data/remote/ResponseProfileDto.kt
package com.example.shoestoreapp.features.user.profile.data.remote

import com.google.gson.annotations.SerializedName

data class ResponseProfileDto(
    @SerializedName("userGuid") val userGuid: String,
    @SerializedName("userName") val userName: String?,
    @SerializedName("email") val email: String?,
    @SerializedName("avatarUrl") val avatarUrl: String?,
    @SerializedName("dateOfBirth") val dateOfBirth: String?
)

